README.md

```diff
@@ -114,9 +114,6 @@ These are small example programs used in our documentation.
 ```
 make example_c
```

Comment:
if you're removing, remove them all - C and C++ examples seems a bit outdated here, and we also have specific examples in theirs repositories.

And please put some comment in this sections that examples were moved to individual repos and can be found there only.